geoutils/misc.py

```diff
@@ -155,75 +154,6 @@ def new_func(*args, **kwargs):  # type: ignore
     return deprecator_func
 
 
-def diff_environment_yml(fn_env: str, fn_devenv: str, print_dep: str = "both") -> None:
```

Comment:
Just thinking, we could also test that the pip and conda environments are the same?

Reply:
I meant checking that the content of `requirement.txt` (or rather `setup.py` I guess?) is consistent with environment.yml. 
I'm mostly concerned about installing from source, which requires a `pip install -e .` command. If any requirement listed in `setup.py` is not in environement.yml, then it would cause pip to install it.